modules/kmm-firmware-support.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: CONCEPT
+[id="kmm-firmware-support_{context}"]
+= KMM firmware support
+
+Kernel modules sometimes need to load firmware files from the filesystem. KMM supports copying firmware files from the
```

Comment:
Per [IBM Word Usage](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#f) for file system:

> Write as shown: two words

```suggestion
Kernel modules sometimes need to load firmware files from the file system. KMM supports copying firmware files from the
```

Reply:
changed.